pontoon/base/models.py

```diff
@@ -30,6 +33,21 @@ def user_display_name(self):
 User.add_to_class('display_name', user_display_name)
 
 
+def user_gravatar_url(self, size):
+    gravatar_url = "//www.gravatar.com/avatar/" + \
```

Comment:
DOUBLE QUOTED STRINGS AAAAAAAAAHHHHHHH (we prefer single quotes unless the string contains single quotes that you don't want to escape)

Also, I'm generally not a fan of line continuations using `\`. I prefer using parenthesis to group statements, but in this case you'd do better to assign the hash to a variable and then using string formatting to insert it in the URL. You could even just do a format at the end to add the query args too instead of using `+=`. These are all nits though, the code itself seems to work fine.